Tests/IntegrationTests/PostgresNIOTests.swift

```diff
@@ -56,6 +56,21 @@ final class PostgresNIOTests: XCTestCase {
         XCTAssertEqual(try rows?.first?.decode(String.self, context: .default).contains("PostgreSQL"), true)
     }
 
+    func testSimpleQueryWithAdditionalParameters() throws {
```

Comment:
please do this in the async way (https://github.com/vapor/postgres-nio/blob/main/Tests/IntegrationTests/AsyncTests.swift). this file will be removed at 2.0 anyway.

Reply:
done